INSTALL/INSTALL.tpl.sh

```diff
@@ -676,9 +788,8 @@ fi
 # TODO: Move support map to top
 
 SUPPORT_MAP="
-x86_64-centos-8
+x86_64-centos-7
```

Comment:
Note: removing support for some centos en RHEL where the script doesn't (and didn't) work before.